THIRD_PARTY_LICENSES.txt

```diff
@@ -8909,7 +8861,47 @@ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR
 IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 =========================================
 END OF minimatch NOTICES AND INFORMATION
+------------------ END OF DEPENDENCY LICENCE --------------------
```

Comment:
It seems like this line has got duplicated. Please remove it, since the END of this dependency license is appearing after all its dependencies. Thanks.
```suggestion
```